packages/@tailwindcss-upgrade/src/template/codemods/important.test.ts

```diff
@@ -0,0 +1,22 @@
+import { __unstable__loadDesignSystem } from '@tailwindcss/node'
+import dedent from 'dedent'
+import { expect, test } from 'vitest'
+import { important } from './important'
+
+let html = dedent
+
+test.each([
+  ['!flex', 'flex!'],
+  ['min-[calc(1000px+12em)]:!flex', 'min-[calc(1000px_+_12em)]:flex!'],
+  ['md:!block', 'md:block!'],
+
+  // Does not change non-important candidates
+  ['bg-blue-500', 'bg-blue-500'],
+  ['min-[calc(1000px+12em)]:flex', 'min-[calc(1000px+12em)]:flex'],
+])('%s => %s', async (candidate, result) => {
+  let designSystem = await __unstable__loadDesignSystem('@import "tailwindcss";', {
+    base: __dirname,
+  })
+
+  expect(important(designSystem, candidate)).toEqual(result)
+})
```

Comment:
Migrated this test to what I found more useful in other migrations. Basically it only tests the migration function and not the text replace logic (we have other tests for this incl. integration tests)